protocol/contracts/interfaces/oud/Oud__ITempleLineOfCredit.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Oud (protocol/contracts/interfaces/oud/IOudtoken.sol)
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {IERC20Permit} from '@openzeppelin/contracts/token/ERC20/extensions/draft-IERC20Permit.sol';
```

Comment:
unused

Reply:
fixed in [1e3705e](https://github.com/TempleDAO/temple/pull/760/commits/1e3705ea583ca5bf20dcefba8b70dda6bc940a5f)